arm/Microsoft.ContainerRegistry/registries/deploy.bicep

```diff
@@ -59,7 +59,7 @@ param retentionPolicyDays int = 15
   'enabled'
 ])
 @description('Optional. The value that indicates whether encryption is enabled or not.')
-param encryptionStatus string = 'disabled'
+param encryptionStatus string = 'enabled'
```

Comment:
Does enabling encryption require a specific feature to be enabled in the Azure subscription? In that case for consistency with other modules this need to be disabled by default.

Reply:
We reeeeaally need this discussion again :D

Reply:
@eriqua : encryption is already part of Bicep code, we just enabling it. Moreover, also gone through with Bicep code doc for RSV, Doesn't seems to be any additional feature require at Azure subscription.

Reply:
@MrMCake I'm not sure I understand your comment. This is what the team decided *so far* (meaning it may change after further discussion) as per issue #1273 and applies to all resources in the same scenario. That doesn't mean we should disable encryption by default. It means we disable it in case something (like a feature) needs to be explicitly enabled on the subscription which is not enabled by default, causing the module deployment to fail otherwise. Hence my question. If that is not the case here, i.e. if no additional feature is required as per @prasanjeets comment, then I'm totally fine to have the value enabled by default.